spec/system/avo/trix_field_spec.rb

```diff
@@ -46,10 +52,33 @@
       it "displays the posts body" do
         visit "/admin/resources/posts/#{post.id}"
 
-        click_on "Show content"
-
+        expect(page).not_to have_link("More content", href: "javascript:void(0);")
         expect(find_field_value_element("body")).to have_text ActionView::Base.full_sanitizer.sanitize(body)
       end
+
+      context "when body has more then 1 line" do
+        let!(:body) do
+          <<~HTML
+        <div>test1</div>
+        <div>test2</div>
+        <div>test3</div>
+          HTML
+        end
+
+        it "displays correct button" do
+          visit "/admin/resources/posts/#{post.id}"
+
+          expect(page).to have_link("More content", href: "javascript:void(0);")
+        end
+
+        it "displays correct button after extended content" do
+          visit "/admin/resources/posts/#{post.id}"
+
+          click_on "More content"
+
+          expect(page).to have_link("Less content", href: "javascript:void(0);")
+        end
```

Comment:
It's possible to speed up the tests by joining those, performance and speed on tests are always great!